conmon/ctr_logging.c

```diff
@@ -156,18 +187,9 @@ static int write_k8s_log(stdpipe_t pipe, const char *buf, ssize_t buflen)
 		/* TODO: We should handle failures much more cleanly than this. */
 		return -1;
 
+	ptrdiff_t line_len = 0;
 	while (buflen > 0) {
-		const char *line_end = NULL;
-		ptrdiff_t line_len = 0;
-		bool partial = FALSE;
-
-		/* Find the end of the line, or alternatively the end of the buffer. */
-		line_end = memchr(buf, '\n', buflen);
-		if (line_end == NULL) {
-			line_end = &buf[buflen - 1];
-			partial = TRUE;
-		}
-		line_len = line_end - buf + 1;
+		bool partial = get_line_len(&line_len, buf, buflen);
```

Comment:
nit of a nit, but I don't think you need to declare this as bool here.

Reply:
Why not? It's a previously undeclared variable :)